pkg/handler/unrouted_handler.go

```diff
@@ -646,6 +646,7 @@ func (handler *UnroutedHandler) HeadFile(w http.ResponseWriter, r *http.Request)
 		} else {
 			resp.Header["Upload-Length"] = strconv.FormatInt(info.Size, 10)
 			resp.Header["Content-Length"] = strconv.FormatInt(info.Size, 10)
+			resp.Header["Accept-Ranges"] = "bytes"
```

Comment:
We may need to check if the underlying store returns a ReadSeeker before advertising this